README.md

```diff
@@ -259,6 +259,7 @@ rustup target add wasm32-unknown-emscripten
 Set up Emscripten via emsdk. We’ll use the incoming version of Emscripten in order to get the best output.
 
 ```shell
+# Make sure to have cmake installed before running this: apt install cmake
```

Comment:
can you provide an optional install command for macOS?

Reply:
What should it be? `brew install cmake`?